builders/ethereum/dev-env/thirdweb.md

```diff
@@ -110,184 +112,206 @@ thirdweb offers SDKs for a range of programming languages, such as React, React
+  id: BigInt(1287),
+  rpc: "https://moonbase-rpc.dwellir.com",
+})
 ```
 
 ### Get Contract {: #get-contract }
 
-To connect to your contract, use the SDK’s [`getContract`](https://portal.thirdweb.com/typescript/sdk.thirdwebsdk.getcontract){target=\_blank} method.
+To connect to your contract, use the SDK’s [`getContract`](https://portal.thirdweb.com/references/typescript/v5/getContract){target=\_blank} method.  As an example, let's fetch data from an [Incrementer contract on Moonbase Alpha](https://moonbase.moonscan.io/address/0xa72f549a1a12b9b49f30a7f3aeb1f4e96389c5d8){target=\_blank}.
 
-```javascript
-import { useContract } from '@thirdweb-dev/react';
+```typescript title="App.tsx"
+import { getContract } from "thirdweb";
+import { client } from "./client";
 
-function App() {
-  const { contract, isLoading, error } = useContract('INSERT_CONTRACT_ADDRESS');
-}
+const myContract = getContract({
+  client,             
+  chain: moonbase,       
+  address: INSERT_CONTRACT_ADDRESS,   
+  abi: INSERT_ABI               
+});
 ```
 
 ### Calling Contract Functions {: #calling-contract-functions }
 
-For extension based functions, use the built-in supported hooks. There are several hooks available for you to use, the following are a few examples:
+To call a contract in the latest version of the SDK, you can use [`prepareContractCall`](https://portal.thirdweb.com/typescript/v5/transactions/prepare){target=\_blank}.
 
-- Use the NFTs extension to access a list of NFTs owned by an address via the [`useOwnedNFTs` hook](https://portal.thirdweb.com/react/react.useownednfts){target=\_blank}:
+```typescript title="App.tsx"
+import { prepareContractCall } from "thirdweb";
 
-    ```javascript
-    import { useOwnedNFTs, useContract, useAddress } from '@thirdweb-dev/react';
+const tx = prepareContractCall({
+          contract,
+          method: "increment",
+          params: [],
+        });
+```
 
-    // Your smart contract address
-    const contractAddress = 'INSERT_CONTRACT_ADDRESS';
+We can trigger this contract call from a ThirdWeb [Transaction button](https://portal.thirdweb.com/typescript/v5/react/components/TransactionButton){target=\_blank}, which has some neat features built in. For example, if you're on the incorrect network, the button will prompt you to switch networks. In the below snippet we'll also add some error handling as a good practice. 
 
-    function App() {
-      const address = useAddress();
-      const { contract } = useContract(contractAddress);
-      const { data, isLoading, error } = useOwnedNFTs(contract, address);
-    }
-    ```
+```typescript title="App.tsx"
+import { TransactionButton } from "thirdweb/react";
+import { prepareContractCall } from "thirdweb";
 
-- Use the [`useContractRead` hook](https://portal.thirdweb.com/react/react.usecontractread){target=\_blank} to call any read functions on your contract by passing in the name of the function you want to use:
+function IncrementButton({ contract }) {
```

Comment:
Getting "Binding element 'contract' implicitly has an 'any' type.ts(7031)" error - can you please double-check?